orttraining/orttraining/test/python/orttraining_test_ortmodule_bert_classifier.py

```diff
@@ -208,25 +208,33 @@ def test(model, validation_dataloader, device, args):
 
 def load_dataset(args):
     # 2. Loading CoLA Dataset
-    print('Downloading dataset...')
+
+    data_dir = '/bert_data/hf_data/glue_data/CoLA/original/raw'
```

Comment:
This breaks local tests on dev machines

Reply:
This will not break local tests. The logic is a waterfall:
1. Check if mount exists at this path ```'/bert_data/hf_data/glue_data/CoLA/original/raw'```. In the case of dev machines, that data does exist, so the test would just use that data.
2. If it does not exist, download to a relative repository and override ```data_dir``` to that relative directory.

Having said that, I could modify this so that we don't use absolute paths. I would need to percolate the need to use the mounted data through an argument to the script then.